listenbrainz/webserver/static/js/src/UserPageHeading.tsx

```diff
@@ -51,7 +51,7 @@ const UserPageHeading = ({
           />
         )}
       </h2>
-      {loggedInUser && (
+      {loggedInUser && user.name !== loggedInUser.name && (
```

Comment:
```suggestion
      {loggedInUser && user?.name !== loggedInUser.name && (
```
 I'm not sure if user can be undefined, but it doesn't hurt to make sure we don't try to access a subproperty of undefined

Reply:
I like the way you code :)